charts/mailu/templates/envvars-configmap.yaml

```diff
@@ -202,6 +201,13 @@ data:
   REDIS_ADDRESS: {{ include "mailu.redis.serviceFqdn" . }}
   SMTP_ADDRESS: {{ include "mailu.postfix.serviceFqdn" . }}
 
+{{- if .Values.front.externalService.enabled }}
+  PROXY_PROTOCOL: {{ include "mailu.proxyProtocolPorts" . | quote }}
+{{- end }}
+
+{{- if .Values.externalDatabase.enabled }}
+  DB_APPENDIX: {{ .Values.externalDatabase.appendix | quote }}
+{{- end }}
```

Comment:
minor nit-pick. The indentation should remain consistent, so please indent the IFs as well.

Reply:
You're absolutely right, I only looked at the `with` above. Sorry for that.